middleware/recover/config.go

```diff
@@ -10,11 +10,25 @@ type Config struct {
 	//
 	// Optional. Default: nil
 	Next func(c *fiber.Ctx) bool
+
+	// EnableStackTrace enables printing stack trace
+	//
+	// Optional. Default: false
+	EnableStackTrace bool
+
+	// PrintHandler defines a function to print stack trace
+	//
+	// Optional. Default: defaultPrintHandler
+	PrintHandler func(e interface{})
 }
 
+var defaultStackTraceBufLen = 1024
+
 // ConfigDefault is the default config
 var ConfigDefault = Config{
-	Next: nil,
+	Next:             nil,
+	EnableStackTrace: false,
+	PrintHandler:     defaultPrintHandler,
```

Comment:
I'm wondering if `StackTraceHandler` is more appropriate? It maybe does not just print the stack trace, but also can store it or send it to remote hosts.

Reply:
Got it. ``StackTraceHandler`` is better than ``PrintHandler ``